services/explorer/tests/api.rs

```diff
@@ -485,28 +510,31 @@ fn test_explorer_transaction_statuses() {
     );
 
     // Now, the same statuses retrieved via explorer web API.
-    let statuses: Vec<_> = [
-        tx.object_hash(),
-        error_tx.object_hash(),
-        panicking_tx.object_hash(),
-    ]
-    .iter()
-    .map(|hash| {
-        let info: TransactionInfo = api
-            .public(ApiKind::Explorer)
-            .query(&TransactionQuery::new(*hash))
-            .get("v1/transactions")
-            .unwrap();
-        info.as_committed().unwrap().status().map_err(Clone::clone)
-    })
-    .collect();
+    let statuses = {
+        let mut output = Vec::new();
```

Comment:
Nit: You can simply write `let mut statuses = Vec::new()` and avoid the additional block here.